bitcoin/src/address/mod.rs

```diff
@@ -2,25 +2,28 @@
 
 //! Bitcoin addresses.
 //!
-//! Support for ordinary base58 Bitcoin addresses and private keys.
+//! Support for segwit and legacy addresses (bech32 and base58 respectively).
 //!
-//! # Example: creating a new address from a randomly-generated key pair
+//! # Examples
+//!
+//! ### Creating a new address from a randomly-generated key pair.
```

Comment:
Why you are going from H1 to H3?
```suggestion
//! # Examples
//!
//! ## Creating a new address from a randomly-generated key pair.
```

Reply:
Because H1 is rendered the same as H2 in rustdocs

Reply:
Unless I'm mistaken

Reply:
Ok, just warned because it seemed like an error. But if it was intentional I don't mind.

Reply:
Cool, thanks man.

Reply:
I rendered the docs and checked the HTML. I usually do that for rustdocs changes but I appreciate you noticing things. Thanks